### PR TITLE
Add conda recipe

### PR DESCRIPTION
**Pull request by k-dominik**
This is the first one (of a few focused) PRs superseeding #12 

added a conda recipe. Couldn't go noarch here because of windows specifics for installation (could have of course got around this with an additional installation package, but I think it's okay having it like this, given it's for distribution only and not used as a library).

### CODE REVIEW

File: conda-recipe/covid-shortcut.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "menu_name": "covid if annotations tool",
+    "menu_items":
+        [
+            {
+                "name": "covif-if",
```
Comment (constantinpape) [inline conda-recipe/covid-shortcut.json]: TYPO: could you change to `covid-if`

File: conda-recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  entry_points:
+    {% for ep in data['entry_points']['console_scripts'] %}
+    - {{ ep }}
+    {% endfor %}
+requirements:
+  build:
+    - python >=3.6
+    - pip
+    # can be removed if bumpversion PR is merged
+    - setuptools_scm
```
Comment (constantinpape) [inline conda-recipe/meta.yaml]: I just merged the bumpversion PR, so can be removed :)

**Review by constantinpape — CHANGES_REQUESTED** [review]
Looks good except for typo and now obsolete dependency.

File: conda-recipe/meta.yaml (hunk 2 above)
Comment (k-dominik) [reply conda-recipe/meta.yaml]: done by rebasing.

File: conda-recipe/covid-shortcut.json (hunk 1 above)
Comment (k-dominik) [reply conda-recipe/covid-shortcut.json]: done, thank you for spotting it!

File: conda-recipe/meta.yaml (hunk 2 above)
Comment (constantinpape) [inline conda-recipe/meta.yaml]: I still see this in the diff. Did something go wrong in the rebase or am I missing something?

Comment (k-dominik) [reply conda-recipe/meta.yaml]: you might have to click on "refresh" on the top of the "files changed" page

Comment (constantinpape) [reply conda-recipe/meta.yaml]: No, I think it's still there:
https://github.com/k-dominik/covid-if-annotations/blob/add-conda-recipe/conda-recipe/meta.yaml#L21-L22

Comment (k-dominik) [reply conda-recipe/meta.yaml]: sorry - that was my mistake. Doing to many things in parallel I have only checked setup.py for the setuptools_scm remainders. Done!

**Pull request merged**